packages/node_modules/@ciscospark/react-component-activity-share-thumbnail/src/index.js

```diff
@@ -17,6 +17,8 @@ const propTypes = {
   file: PropTypes.shape({
     displayName: PropTypes.string,
     fileSize: PropTypes.number,
+    image: PropTypes.shape(),
```

Comment:
This should be object.
```suggestion
    image: PropTypes.object(),
```

Reply:
I think I originally made this a shape since, I believe, there are two properties associated with image, `height` & `width`, and added them into the proptype, however wasnt sure if height & width were being passed as strings or numbers

Reply:
Right, but a shape prop type indicates that you will provide an actual shape. `shape()` doesn't tell us anything about the property, so let's just use the ambiguous `.object`